planet.php

```diff
@@ -0,0 +1,68 @@
+<?php
+require 'vendor/autoload.php';
+
+include 'src/Planet.php';
+use PlanetApp\PlanetHydrator;
+
+$dsn = 'mysql:host=db; dbname=planet_collection';
+$username = 'root';
+$password = 'password';
+
+if(!isset($_GET['planetId'])){header('Location: ./index.php');}
```

Comment:
put the header on a new line

Reply:
done

Reply:
put this on line 6

Reply:
great suggestion - will improve redirect speed